tests/test_backtrace.rs

```diff
@@ -69,24 +82,28 @@ pub mod structs {
     }
 
     #[test]
+    #[cfg_attr(not(feature = "stdbacktrace"), allow(unstable_name_collisions))]
```

Comment:
This wouldn't be needed if the `thiserror::Backtrace::bracktrace` method was renamed. Opinions on what it should be named are welcome.